_data/events.json

```diff
@@ -0,0 +1,321 @@
+[
+   {
+      "name":"AMTA 2024",
+      "id": "amta-2024",
+      "description": "16th biennial conference of the Association for Machine Translation in the Americas",
+      "startDate":"2024-09-30",
+      "endDate":"2024-10-02",
+      "location":{
+         "type":"VirtualLocation",
+         "name":"Online",
+         "eventAttendanceMode": "OnlineEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Organization",
+         "name":"Association for Machine Translation in the Americas",
+         "url":"https://amtaweb.org"
+      },
+      "futureTenseOpeningParagraph": "The sixteenth Biennial Conference of the Association for Machine Translation in the Americas (<strong><a href='/amta'>AMTA</a> 2024</strong>) will be hosted in Chicago, Illinois from 30 September to 2 October, 2024. <br><br> Workshops and tutorials will take place earlier, mid-September. The date is yet to be announced.",
+      "pastTenseOpeningParagraph": "The sixteenth Biennial Conference of the Association for Machine Translation in the Americas (<strong><a href='/amta'>AMTA</a> 2024</strong>) was hosted in Chicago, Illinois from 30 September to 2 October, 2024.",
+      "importantDates": [
+         {
+            "name": "Submission deadline",
+            "date": "6 June "
+         },
+         {
+            "name": "Notification of acceptance",
+            "date": "18 July "
+         },
+         {
+            "name": "Final “camera-ready” papers for proceedings",
+            "date": "16 August"
+         }
+      ]
+   },
+   {
+      "name":"AmericasNLP 2024",
+      "id": "americasnlp-2024",
+      "description": "Machine translation for indigenous languages of the Americas",
+      "startDate":"2024-06-20",
+      "endDate":"2024-06-21",
+      "location":{
+         "type":"Place",
+         "name":"Mexico City, Mexico",
+         "eventAttendanceMode": "OfflineEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Person",
+         "url":"http://turing.iimas.unam.mx/americasnlp/st.html"
+      },
+      "futureTenseOpeningParagraph": "The <strong>Fourth AmericasNLP Competition</strong> will take place online in June, at NAACL 2024 in Mexico City. The competition focused on creating machine translation systems for indigenous languages from the Americas.",
```

Comment:
Ideally we'd avoid this.

Also should we use Markdown instead of HTML?

Reply:
Should we just remove future/past tenses and only keep ```opening_paragraph```? But this way, it should be modified manually when the event happens.

Well, each event page is rendered through HTML. I don't think there is a straightforward way to use Markdown syntax inside HTML?